demoqa.testing/src/test/java/controlExtensions/MultiSelect.java

```diff
@@ -0,0 +1,45 @@
+package controlExtensions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class MultiSelect extends ControlExtensions implements IMultiSelect{
+
+	public MultiSelect(WebElement mappedElement) {
+		super(mappedElement);
+	}
+
+	@Override
+	public void selectOptions(int selections, WebDriver driver) {
+		//WebDriverWait wait = new WebDriverWait(driver, 1);
+		//wait.until(ExpectedConditions.elementToBeClickable(mappedElement));
+		driver.switchTo().defaultContent();
```

Comment:
Should not be needed

Reply:
line 24 switches to the default content so the iframe element does not intercept the click

Reply:
> Should not be needed

line 24 switches to the default content so the iframe element does not intercept the click